helpers/configHelpers.js

```diff
@@ -32,6 +32,34 @@ function deleteRemovedKeys(dc, cc) {
   }
 }
 
+/**
+ * takes two (partial) webpack plugin configs and compares if a key was overwritten in custom config to then merge the otions to one plugin initialization
+
+ * @param {object} dc default configuration in current environment
+ * @param {object} cc custom configuration from project in current environment
+ * @returns {object, object} Default an Custom Configs with priority to custom config
+ */
+function mergeConfigPlugins(dc, cc) {
+  let defaultConf = dc;
+  let customConf = cc;
+
+  if (customConf.plugins) {
+    defaultConf.plugins =
+      defaultConf.plugins.map((plugin) => {
+        const customPlugin = customConf.plugins.find(
+          (customPlugin) => customPlugin.constructor.name === plugin.constructor.name
+        );
+        if (customPlugin) {
+          return Object.assign(plugin, customPlugin);
+        }
+        return plugin;
+      })
+
+      delete customConf.plugins;
+  }
+  return {defaultConf, customConf};
+}
```

Comment:
merges plugins von `customConf` in `defaultConf` und löscht anschließend `customConf.plugins`, sodass diese nur 1x initialisiert werden.

Reply:
This would make an awesome code comment :)

Reply:
Altered the comment above